Makefile

```diff
@@ -1,11 +1,16 @@
 run:
-	@python hr-data-analytics-package-folder/main.py
+	@python hr_data_analytics_package_folder/main.py
 
 run_uvicorn:
-	@uvicorn hr-data-analytics-package-folder.api:app --reload
+	@uvicorn hr_data_analytics_package_folder.api:app --reload
 
 install:
 	@pip install -e .
 
 test:
 	@pytest -v tests
+
+
+reset_dummy_data_files:
+	@rm -rf ${DUMMY_DIR}
+	@mkdir  ${DUMMY_DIR}
```

Comment:
Make command to reset a local directory which stores the dummy data to use for presentation.